Update README.md (#4)
DevExpress-Examples/blazor-grid-use-a-dictionary-to-configure-the-component: README.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T907025)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Grid for Blazor - How to use a dictionary to configure the component state

> This example demonstrates how you can use a dictionary to configure the [Blazor Grid](https://docs.devexpress.com/Blazor/403143/components/grid) component. This approach allows you to configure multiple components based on one set of settings.

Reviewer: 1. Approach.
2. Set of settings | or settings set, for that matter. It does not sound very well.

```suggestion
This example demonstrates how you can use a dictionary to configure the [Blazor Grid](https://docs.devexpress.com/Blazor/403143/components/grid) component. This technique allows you to configure multiple components based on one options set.
```

![Use a Dictionary to Configure the Component State](/image.png)

## Overview

> Create a [ComponentBase](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.componentbase?view=aspnetcore-3.1) class descendant that accepts data source, grid columns, and grid settings stored in a dictionary. In the created class, configure the Grid component at runtime as follows:

Reviewer 2: ```suggestion
Create a [ComponentBase](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.componentbase?view=aspnetcore-3.1) class descendant that accepts a data source, grid columns, and grid settings stored in a dictionary. In the created class, configure the Grid component at runtime as follows:
```

> Create a descendant of the [ComponentBase](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.componentbase?view=aspnetcore-3.1) class that accepts data source, grid columns, and grid settings stored in a dictionary. In the class, configure the Grid component at runtime as follows:

Reviewer: Optimization.

```suggestion
Create a [ComponentBase](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.componentbase?view=aspnetcore-3.1) class descendant that accepts data source, grid columns, and grid settings stored in a dictionary. In the created class, configure the Grid component at runtime as follows:
```

}
```

> Create a dictionary that stores setting names and values. Assign the dictionary to the corresponding component parameter to create and configure the Grid:

Reviewer 2: ```suggestion
Create a dictionary that stores setting names and values. Assign this dictionary to the corresponding component parameter to create and configure the Grid:
```
